task_10/index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
+    <link rel="stylesheet" type="text/css" href="css/reset.css">
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+    <script type="text/javascript" src="js/config.js"></script>
+    <script type="text/javascript" src="js/helper.js"></script>
+    <script type="text/javascript" src="js/models/model.js"></script>
+    <script type="text/javascript" src="js/views/view.js"></script>
+    <script type="text/javascript" src="js/services/weather_service.js"></script>
+    <script type="text/javascript" src="js/services/storage_service.js"></script>
+    <script type="text/javascript" src="js/services/list_service.js"></script>
+    <script type="text/javascript" src="js/services/favorites_service.js"></script>
+    <script type="text/javascript" src="js/services/history_service.js"></script>
+    <script type="text/javascript" src="js/controllers/controller.js"></script>
+    <script type="text/javascript" src="js/app.js"></script>
+    <title>Weather App</title>
+  </head>
+  <body>
+    <header class="header">
+      <div class="flex-container">
+          <span>
+            <button class="btn" id="add-favorite-btn" title="Adds city to favorites">
+              <i class="fa fa-star" aria-hidden="true"></i>
+            </button>
+          </span>
+          <span class="loc-field">
+            <input class="btn wide" id="loc-field" type="text" placeholder="Enter location..." title="Enter location here">
+          </span>
+          <span class="base-units">
+            <select class="btn" id="base-units" title="Selects metric or imperial units">
```

Comment:
It's okay but how many options we may have? Two? Maybe three? This is radio buttons, isn't it?

Any element you use would work but think about user experience more.

Reply:
I've already used radio buttons there before, but (IMHO) select is more convenient.

Reply:
From mathematical perspective id cannot me more convenient because it require 2 actions instead of 1. Of course, ratio buttons should be done in right way to be really convenient.

Anyway, keep in mind I am just a reviewer, and you are product owner. Lets leave select, it's good!

Reply:
So am I - just a product owner, who tries to negotiate some disputable issues with reviewer. :grinning: I'll try to pay due attention to using appropriate UI elements.